packages/server/src/integrations/googlesheets.ts

```diff
@@ -395,7 +397,12 @@ class GoogleSheetsIntegration implements DatasourcePlus {
           sheet,
         })
       case Operation.CREATE_TABLE:
-        return this.createTable(json?.table?.name)
+        if (json.table === undefined) {
```

Comment:
Any reason on why excluding null checks?

Reply:
No, I'll fix this. 🙏